tests/src/Common/test_dependencies/test_dependencies.csproj

```diff
@@ -22,6 +22,9 @@
     <PackageReference Include="System.Security.Permissions">
       <Version>$(CoreFxPackageVersion)</Version>
     </PackageReference>
+    <PackageReference Include="System.Runtime.Intrinsics.X86">
+      <Version>4.5.0-preview1-25718-03</Version>
```

Comment:
@mellinoe can I set a more flexible package version here?

Reply:
I think you can just use this like the one above:

`<Version>$(CoreFxPackageVersion)</Version>`

That property will get automatically updated every so often and the package will get upgraded.

Reply:
Last week this property did not work, but today it works fine... Thank you!